sql/core/src/test/resources/sql-tests/results/date.sql.out

```diff
@@ -331,12 +331,20 @@ struct<date_add(DATE '2011-11-11', 1):date>
 
 
 -- !query
-select date_add(timestamp_ltz'2011-11-11 12:12:12', 1), date_add(timestamp_ntz'2011-11-11 12:12:12', 1)
+select date_add(timestamp_ltz'2011-11-11 12:12:12', 1)
+-- !query schema
+struct<date_add(TIMESTAMP '2011-11-11 12:12:12', 1):date>
+-- !query output
+2011-11-12
+
+
+-- !query
+select date_add(timestamp_ntz'2011-11-11 12:12:12', 1)
```

Comment:
if `timestamp_ntz` becomes the default in the future, it's better to support this query as well to not break anything.

Reply:
The LTZ one will fail in ANSI mode. We tend to make the NTZ behavior consistent in default/ANSI mode, right?

Reply:
It's a different topic. Breaking change means it's very hard to enable NTZ by default in the future.

Reply:
I think we tend to make LTZ/NTZ behavior consistent within default mode, or within ANSI mode, but not between default and ANSI mode.

Reply:
I will have a follow-up for NTZ behavior.